src/geometry/ramer_douglas_peucker.rs

```diff
@@ -0,0 +1,86 @@
+use crate::geometry::Point;
+
+pub fn ramer_douglas_peucker(points: &[Point], epsilon: f64) -> Vec<Point> {
+    if points.len() < 3 {
+        return points.to_vec();
+    }
```

Comment:
Please add tests, where you call `ramer_douglas_peucker` with empty, single point and two point inputs.